node-db/lesson1.md

```diff
@@ -135,6 +135,42 @@ At the endo of the wizard, you should see a new file called `package.json` in yo
 
 Here is an example `package.json` file for a project called [Passport](https://github.com/jaredhanson/passport/blob/master/package.json).
 
+# What is JSON?
```

Comment:
I think the topic should be more or less explained at a point when it is introduced, and this is when we talk about package.json